modules/executors/variables.tf

```diff
@@ -27,7 +27,7 @@ variable "machine_image" {
 
 variable "machine_type" {
   type        = string
-  default     = "n1-standard-4" // 4 vCPU, 15GB
+  default     = "c2-standard-8" // 8 vCPU, 32GB
```

Comment:
see comments at the end of the diff - this is the default we use everywhere, so adjusted it in here, too.